components/developer/golang-16/golang-16.p5m

```diff
@@ -25,8 +26,8 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 <transform file path=usr/lib/golang/1.6/pkg/tool/solaris_$(MACH64)/.+ -> \
     default mode 0555>
 
-link path=usr/bin/go target=../lib/golang/1.6/bin/go
-link path=usr/bin/gofmt target=../lib/golang/1.6/bin/gofmt
+link path=usr/bin/go target=../lib/golang/1.6/bin/go mediator=golang mediator-version=1.6
```

Comment:
COMPONENT_REVISION has to be bumped in the Makefile, so the new package is published. Otherwise, golang-18 won't be installable and will fail with conflicting actions error.